src/vs/workbench/parts/terminal/electron-browser/terminal.contribution.ts

```diff
@@ -42,6 +42,26 @@ configurationRegistry.registerConfiguration({
 			'description': nls.localize('terminal.integrated.fontFamily', "The font family used by the terminal (CSS font-family format)."),
 			'type': 'string',
 			'default': 'Menlo, Monaco, Consolas, "Droid Sans Mono", "Courier New", monospace, "Droid Sans Fallback"'
+		},
+		'terminal.integrated.fontSize': {
+			'description': nls.localize('terminal.integrated.fontSize', "The font size used by the terminal (CSS font-size format)."),
+			'type': 'string',
+			'default': ''
+		},
+		'terminal.integrated.lineHeight': {
+			'description': nls.localize('terminal.integrated.lineHeight', "The line height used by the terminal (CSS line height format)."),
+			'type': 'string',
+			'default': ''
+		},
+		'terminal.integrated.charWidth': {
```

Comment:
I think https://github.com/Microsoft/vscode/issues/6456 should be figured out before the font size and line height is added in order to prevent addition the `charWidth`/`charHeight` settings.

Also font size and line height will potentially be part of UI themes (#76) as well, we would accept this though if it worked without a char width/height setting due to the immediate accessibility need.